External/ProcessEx.cs

```diff
@@ -308,6 +308,42 @@ public void SetString(PointerEx absoluteAddress, string Value)
             SetArray(absoluteAddress, Value.Bytes());
         }
 
+        public PointerEx OffsetCalculator(string moduleName, int baseOffset, int[] offsets)
+        {
+            PointerEx moduleAddr = GetLoadedModuleAddress(moduleName);
+            PointerEx result = moduleAddr + baseOffset;
+
+            byte[] buffer = new byte[IntPtr.Size];
+            foreach (int offset in offsets)
+            {
+                if (IntPtr.Size == 4)
+                {
+                    result = BitConverter.ToInt32(GetBytes(result, buffer.Length), 0);
+                }
+                else
+                {
+                    result = BitConverter.ToInt64(GetBytes(result, buffer.Length), 0);
+                }
+
+
+                result = IntPtr.Add(result, offset);
+            }
+
+            return result;
+        }
+
+        public void NOP(PointerEx absolutionAddress, int numberOfBytes)
+        {
+            List<byte> list = new List<byte>();
+            for (int i = 0; i < numberOfBytes; i++)
+            {
+                list.Add(144);
```

Comment:
```suggestion
                bytes[i] = 0x90;
```
Better indicative of the byte being the 0x90 opcode most people are used to.